### DISCUSSION TITLE
Please test stopping a garage door via the web app

### DISCUSSION THREAD

**Discussion by brutella** [brutella/hkknx-public#356] [General]
In the pre-release [3.1.0-b9.3](https://github.com/brutella/hkknx-public/releases/tag/3.1.0-b9.3) there is now a new characteristic to stop a moving garage door. Even though this is not a feature in HomeKit, the web app shows a user interface for this.
<img width="495" alt="Screenshot 2024-11-08 at 12 37 48" src="https://github.com/user-attachments/assets/0d28040c-5415-4b77-a2d1-312f076fde99">

Could somebody please test this feature and verify that it works as expected. Thanks.

**Comment by Motl1** [comment]
Funktioniert!
